wireless_modules/config.example.py

```diff
@@ -1,11 +0,0 @@
-ESSID = "<WIFI NETWORK NAME HERE>"
```

Comment:
Is this file meant to be deleted? It looks like it is meant to be kept as a reference to me (might have been renamed to make the actual config file at one point, which git may interpret as removing this one).